code/game/objects/machinery/doors/poddoor.dm

```diff
@@ -337,3 +337,47 @@ GLOBAL_LIST_INIT(faction_to_campaign_door_signal, list(
 
 /obj/machinery/door/poddoor/nt_lockdown/red
 	code_color = MISSION_CODE_RED
+
+// urban Shutters
+/obj/machinery/door/poddoor/urban
+	icon = 'icons/obj/structures/prop/urban/urbanshutters.dmi'
+	icon_state = "almayer_pdoor1"
+	base_icon_state = "almayer_pdoor"
+	openspeed = 4
+
+/obj/machinery/door/poddoor/urban/open_shutters/Initialize()
+	. = ..()
+	if(opacity)
+		set_opacity(0)
```

Comment:
Why not just set opacity (and density) on the type? Why during init? Also this needs to be under the obj definition

Reply:
Opacity is already set on type, so I don't know why CM did this. All I can think of is the author wanted insurance in case somebody var-edited a shutter to not have opacity for some reason? I removed it anyway.